shared/settings.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+enum DeviceType {
+    RM1,
+    RM2,
+};
```

Comment:
```suggestion
enum DeviceType {
    Unknown,
    RM1,
    RM2,
};
```

Reply:
Why add Unknown if it's never used for now?

Reply:
https://github.com/Eeems/oxide/pull/108#discussion_r531171304 <- It'll be used as part of that change